js/subtitles.js

```diff
@@ -117,7 +117,16 @@
             subtitles.innerHTML = 'Unable to connect due to network issue or unable to gain microphone access.';
         }, 15000);
 
-        recognizer.startContinuousRecognitionAsync();
+        var userAgent = navigator.userAgent.toLowerCase();
+        if (userAgent.indexOf(' electron/') > -1) {
```

Comment:
The idea here is not to break the current functionality when the page is hosted in Electron. The user interaction requirement is only enforced when hosted in the browser sandbox.